gfop/get_food_counts.py

```diff
@@ -83,14 +84,15 @@ def get_file_food_counts(gnps_network, sample_types, all_groups, some_groups,
     sample_counts = sample_types_selected.value_counts()
     sample_counts_valid = sample_counts.index[sample_counts > water_count]
     sample_types_selected = sample_types_selected[
-        sample_types_selected.isin(sample_counts_valid)]
+        sample_types_selected.isin(sample_counts_valid.get_level_values(level=0))]
```

Comment:
The MultiIndex is because we're doing `.value_counts()` on a DataFrame now, which returns the counts with a MultiIndex. Previously [`sample_types` was a Series](https://github.com/ka-west/gfop/blob/main/gfop/get_food_counts.py#L74).

See below for a fix. If that is changed the `.get_level_values` in this line and `.reset_index` in the next line can be removed.